projects/vdk-plugins/vdk-ipython-ext/README.md

```diff
@@ -0,0 +1,53 @@
+# ipython-ext
+
+Ipython extension for VDK
+
+This extension introduces a magic command for Jupyter.
+The command enables the user to load job_input for his current data job and use it freely while working with Jupyter.
+
+See more about magic commands: https://ipython.readthedocs.io/en/stable/interactive/magics.html
+
+
+## Usage
+To use the extension it must be firstly installed with pip as a python package.
+Then to load the extension in Jupyter the user should use:
+```
+%reload_ext vdk_ipython_ext
```

Comment:
would it not look a bit nicer if it was just `%reload_ext vdk`

Reply:
I am note really sure whether it is ok to name it the same wayt
@tozka can I get your opinion, too?

Reply:
In this way I get some errors with the imports and I thing it would be good to have a little difference since it is not loading the whole vdk but only a small part of it